<ui version="4.0" >
 <class>GlobalGraphViewOptions</class>
 <widget class="QWidget" name="GlobalGraphViewOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>376</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="Camera" >
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QGroupBox" name="groupBox_8" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="title" >
        <string>Control for 3D Movements</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>5</number>
        </property>
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>5</number>
        </property>
        <property name="bottomMargin" >
         <number>5</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="2" column="1" >
         <widget class="QComboBox" name="comboBoxCamera3D_4" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBoxCamera3D" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="comboBoxCamera3D_7" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_21" >
          <property name="text" >
           <string>Left Button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="label_18" >
          <property name="text" >
           <string>Right Button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_20" >
          <property name="text" >
           <string>Middle Button</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QComboBox" name="comboBoxCamera3D_8" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QComboBox" name="comboBoxCamera3D_5" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QComboBox" name="comboBoxCamera3D_2" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QComboBox" name="comboBoxCamera3D_9" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QComboBox" name="comboBoxCamera3D_6" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QComboBox" name="comboBoxCamera3D_3" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_19" >
          <property name="text" >
           <string>Control-Key ^</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_22" >
          <property name="text" >
           <string>Shift-Key ^</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetCameraDefault" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Reset Defaults</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox_2D" >
       <property name="title" >
        <string>Control for 2D Movements</string>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>5</number>
        </property>
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>5</number>
        </property>
        <property name="bottomMargin" >
         <number>5</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="2" column="1" >
         <widget class="QComboBox" name="comboBoxCamera2D_4" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBoxCamera2D" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="comboBoxCamera2D_7" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_23" >
          <property name="text" >
           <string>Left Button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="label_24" >
          <property name="text" >
           <string>Right Button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_25" >
          <property name="text" >
           <string>Middle Button</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QComboBox" name="comboBoxCamera2D_8" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QComboBox" name="comboBoxCamera2D_5" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QComboBox" name="comboBoxCamera2D_2" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3" >
         <widget class="QComboBox" name="comboBoxCamera2D_9" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QComboBox" name="comboBoxCamera2D_6" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QComboBox" name="comboBoxCamera2D_3" >
          <property name="minimumContentsLength" >
           <number>0</number>
          </property>
          <property name="duplicatesEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_26" >
          <property name="text" >
           <string>Control-Key ^</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_27" >
          <property name="text" >
           <string>Shift-Key ^</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
